src/ts3init_match.c

```diff
@@ -141,7 +141,7 @@ static inline __u8* get_payload(const struct sk_buff *skb, const struct xt_actio
 static int calculate_cookie(const struct sk_buff *skb, const struct xt_action_param *par, 
                        struct udphdr *udp, __u64 k0, __u64 k1, __u64* out)
 {
-    switch (par->family)
+    switch (xt_family(par))
```

Comment:
syntax changed with version 4.10.0, please add a version ifdef like in compat_xtables.h

Reply:
done